_includes/manuals/2.1/3.6_upgrade.md

```diff
@@ -165,7 +165,7 @@ To apply these changes, run the installer again without options
 
 #### Step 4 - Restart
 
-Restart the application server (e.g. mongrel, thin, passenger).
+Restart the application server (e.g. puma, passenger).
```

Comment:
Can you remove passenger from here to align with the rest of the changes? We shouldn't be advertising what we don't recommend.

Reply:
Passenger is still supported and there is an installer option to use it.

My bigger issue is that the instructions below it (to restart Apache) are incorrect for Puma. It should really be `systemctl restart foreman`.